feconf.py

```diff
@@ -583,11 +593,14 @@ def get_empty_ratings():
 SITE_LANGUAGE_DATA_URL = '/save_site_language'
 SIGNUP_DATA_URL = '/signuphandler/data'
 SIGNUP_URL = '/signup'
+SKILL_EDITOR_URL_PREFIX = '/skill_editor/create'
```

Comment:
This is premature, only add it when you do the controllers (and at that point, follow existing naming conventions). Ditto below.

Reply:
Done, removed them for now.